core/src/main/java/jenkins/install/InstallState.java

```diff
@@ -208,6 +241,9 @@ public void initializeState() {
             super("DOWNGRADE", true);
         }
         public void initializeState() {
+            // Schedule an update of the update center after a Jenkins downgrade
+            UpdateCenter.updateDefaultSite();
```

Comment:
this does cause startup to block for approx 4 seconds on my machine.

I did try `Jenkins.get().getUpdateCenter().doInvalidateData();` from:
https://github.com/jenkinsci/jenkins/pull/5257 but it wasn't enough, the update timestamp just changed to N/A but the data was never reloaded until the user clicked check now

There's probably a better way to do this though, would love input here.

Reply:
Wrap in `Timer.get`?

Reply:
thanks that sorted that